crates/core/protobuf/node_ctl_svc.proto

```diff
@@ -22,15 +22,16 @@ service NodeCtlSvc {
   rpc GetMetadata(GetMetadataRequest) returns (GetMetadataResponse);
 
   // Provision the Restate cluster on this node.
-  rpc ProvisionCluster(ProvisionClusterRequest) returns (ProvisionClusterResponse);
+  rpc ProvisionCluster(ProvisionClusterRequest)
+      returns (ProvisionClusterResponse);
 }
 
 message ProvisionClusterRequest {
   bool dry_run = 1;
   // if unset then the configured cluster num partitions will be used
   optional uint32 num_partitions = 2;
   // if unset then the configured cluster placement strategy will be used
-  optional restate.cluster.ReplicationStrategy placement_strategy = 3;
+  optional restate.cluster.ReplicationProperty partition_replication = 3;
```

Comment:
You might want to leave a comment about the subtle details. If ReplicationProperty is set, but its set to an empty string, then it mean everywhere replication. This has different meaning from not setting partition_replication here.

Reply:
I updated the comment. Also if an invalid replication property is specified (for example an empty string) the grpc api will return a bad request error. Either not provide it at all, or provide a valid replication property string

Reply:
Good.